tests/comm/test_mnnvl_moe_alltoall.py

```
@@ -709,8 +782,27 @@ def moe_a2a_dispatch_moe_combine_test_impl(distribution, top_k):
+        lora_ids = torch.randint(
+            0,
+            NUM_LORA_ADAPTERS,
+            (local_num_tokens,),
+            dtype=torch.int32,
+            device="cuda",
+        )
+        lora_weights = create_lora_weights(
+            NUM_LORA_ADAPTERS, hidden_size, "cuda", dtype=torch.bfloat16
+        )
+
     payloads, expert_id_payload_index = make_bfloat16_payloads(
```

Ruff (0.15.20)

[warning] Unpacked variable `expert_id_payload_index` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)